app/routes/circulars._archive._index/route.tsx

```diff
@@ -284,35 +297,45 @@ export default function () {
           </Button>
         </Form>
 
-        {query && !isGroupView && (
-          <SortSelector form={formId} defaultValue={sort} />
-        )}
-
         {synonymFlagIsOn && (
-          <Link
-            to={`/circulars?view=${viewState.toLowerCase()}&limit=${limit}`}
-            preventScrollReset
-          >
-            <Button
-              type="button"
-              className="padding-y-1"
-            >{`${viewState} View`}</Button>
-          </Link>
+          <ButtonGroup type="segmented">
+            <Link
+              to={`/circulars?view=index&limit=${limit}`}
+              style={{ textDecoration: 'none' }}
```

Comment:
We haven't needed this in the other `<Link>` components with `className` set to `usa-button`. Does the link get underlined if you remove this?